configs/mmdet3d/mono-detection/mono-detection_static.py

```diff
@@ -0,0 +1,5 @@
+_base_ = ['../../_base_/onnx_config.py']
+codebase_config = dict(
+    type='mmdet3d', task='MonoDetection', model_type='end2end')
+onnx_config = dict(
+    input_names=['imgs'], output_names=['cls_score', 'bbox_pred'])
```

Comment:
better change `imgs` ad `input` as it is mostly used.

Reply:
ok